custom_components/tuya_local/devices/diivoo_dwv010.yaml

```diff
@@ -44,21 +44,20 @@ secondary_entities:
         unit: "l"
   - entity: sensor
     class: battery
-    name: Battery
     category: diagnostic
     dps:
       - id: 7
         type: integer
         name: sensor
         unit: "%"
-  - entity: sensor
+  - entity: attribute
```

Comment:
To add an extra attribute to an entity, you just need to list it under the entity it should be an attribute of. These two dps are already listed under the valves above, so the definitions here can be removed.